Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -1014,18 +1009,6 @@ public final class ManifestLoader: ManifestLoaderProtocol {
         return cmd
     }
 
-    /// Returns the runtime path given the manifest version and path to libDir.
-    private func runtimePath(for version: ToolsVersion) -> AbsolutePath {
-        let manifestAPIDir = self.toolchain.swiftPMLibrariesLocation.manifestAPI
-        if localFileSystem.exists(manifestAPIDir) {
-            return manifestAPIDir
-        }
-
-        // FIXME: how do we test this?
-        // Fall back on the old location (this would indicate that we're using an old toolchain).
-        return self.toolchain.swiftPMLibrariesLocation.manifestAPI.parentDirectory.appending(version.runtimeSubpath)
```

Comment:
are we loosing this fall back?

Reply:
Yep, I talked to @abertelrud about it earlier, this was temporary and is no longer needed at this point.

Reply:
Indeed, this was for the `4` vs `4_2` subpath which is thankfully far behind us at this point (within a single toolchain).